datafusion/core/Cargo.toml

```diff
@@ -107,6 +107,7 @@ doc-comment = "0.3"
 env_logger = "0.9"
 parquet-test-utils = { path = "../../parquet-test-utils" }
 rstest = "0.15.0"
+test-case = "*"
```

Comment:
I would prefer not to add a new dependency -- especially since the tests are simple and we already have `rstest` https://docs.rs/rstest/latest/rstest/ which also has fixtures

I would be happy to hear other opinions as well from other maintainers